packages/shared/src/hooks/useProfileForm.ts

```diff
@@ -30,9 +30,14 @@ export interface ProfileFormHint {
   bluesky?: string;
 }
 
+export type UserFlagsPublic = {
+  showPlusGift: boolean;
+};
```

Comment:
```suggestion
export type UserFlagsPublic = Partial<{
  showPlusGift: boolean;
}>;
```

flags are always optional so I think we can jsut do this on the type

Reply:
then below `Partial` can be removed